cep-9999.md

```diff
@@ -0,0 +1,165 @@
+
+Each `run_exports` metadata `dict` can contain the following fields; each field accepts a list of strings (conda-build specs).
+
+- `weak`
+- `strong`
+- `weak_constrains`
+- `strong_constrains`
+- `noarch`
+
+```json
+{   
+    "info": {
+        "platform": "string",
+        "arch": "string",
+        "subdir": "string",
+        "version": 0
+    },
+    "packages": {
+        "package-version-build.tar.bz2": {
+            "run_exports": {
+                "noarch": [
+                    "string",
+                ],
+                "strong": [
+                    "string",
+                ],
+                "strong_constrains": [
+                    "string",
+                ],
+                "weak": [
+                    "string",
+                ],
+                "weak_constrains": [
+                    "string",
+                ],
+            }
+        }
+    },
+    "packages.conda": {
+        "package-version-build.conda": {
+            "run_exports": {
+                "noarch": [
+                    "string",
+                ],
+                "strong": [
+                    "string",
+                ],
+                "strong_constrains": [
+                    "string",
+                ],
+                "weak": [
+                    "string",
+                ],
+                "weak_constrains": [
+                    "string",
+                ],
+            }
+        }
+    }
+}
+```
+
+If a package does not define `run_exports`, the corresponding entry in `packages` or `packages.conda` will be an empty `run_exports` item:
+
+```json
+{
+    "info": {
+        "platform": "string",
+        "arch": "string",
+        "subdir": "string",
+        "version": 0
+    },
+    "packages": {
+        "package-version-build.tar.bz2": {
+            "run_exports": {}
+        }
+    },
+    "packages.conda": {
+        "package-version-build.conda": {
+            "run_exports": {}
+        }
+    }
+}
+```
+
+> See the validation schema draft in [`run_exports.schema.json`](https://github.com/conda/schemas/pull/25).
+
+
+## Backwards compatibility
+
+This is a new feature, so there is no backwards compatibility to worry about.
+
+## Alternatives
+
+We could maintain the status quo and ask downstream infrastructure to maintain their own database. However, this is a burden on them, and it is trivial to generate this data.
+
+We also considered _adding_ the `run_exports` metadata to `repodata.json`, but this has a few shortcomings:
+
+- It would require extending the `repodata` schema, currently not formally standardized.
+- It would increase the size of the already heavy `repodata.json` files.
+- (Typed) repodata parsers would need to be updated to handle the new field.
+
+Finally, we studied whether the `run_exports` metadata already present in the `channeldata.json` metadata would be enough. However, this metadata is only presented per version (not build), so it is insufficient and incomplete.
```

Comment:
Should we add a sentence saying extending the `channeldata.json` format to add per-build information risks breaking conda-build's `--use-channeldata` option? (We should have a separate discussion about deprecating the `--use-channeldata` option.)